lua/def.c

```diff
@@ -597,7 +597,11 @@ const upb_FileDef* lupb_FileDef_check(lua_State* L, int narg) {
 
 static int lupb_FileDef_Dependency(lua_State* L) {
   const upb_FileDef* f = lupb_FileDef_check(L, 1);
+#if LUA_VERSION_NUM >= 501
```

Comment:
Since we only support lua_version > 501, you can just drop this #if and only keep the corresponding case, and then the same below on lines 618 and 636 (and similarly in those cases use lupb_checkint32 since _checkinteger won't be defined on 5.1)

Reply:
Thanks and fixed.